### ISSUE TITLE
(feature request) Clear description of what conditions create which address-colours in README (and/or legend)

### ISSUE THREAD

**Issue by lyricnz** [LukePrior/nbn-upgrade-map#318]
Current is counter-intuitive: committed addresses with dates mid next year are showing "red".

**Comment by LukePrior** [comment]
Good idea, I'm also open to reworking the legend as I don't think it's the clearest right now and some edge cases exist.

here is the current flow:

is tech='FTTP': FTTP
is tech_change_status='Eligible To Order' && (tech='FTTN' || 'FTTC'): FTTP Upgrade
is (tech_change_status='Build Finalised' || 'MDU Complex Eligible To Apply' || 'MDU Complex Premises In Build') && (tech='FTTN' || 'FTTC'): FTTP Soon
is target_eligibility_quarter < 3 months from file generated && (tech='FTTN' || 'FTTC'): FTTP Soon
is target_eligibility_quarter not provided (historical records) &&(upgrade='FTTP_SA': FTTP Upgrade
is target_eligibility_quarter not provided (historical records) &&(upgrade='FTTP_NA': FTTP Soon
finally simply match to corresponding current technology

**Comment by lyricnz** [comment]
Feels like an committed upgrade >3 months away should be some kind of green, rather than red...

<img width="366" alt="image" src="https://github.com/LukePrior/nbn-upgrade-map/assets/122371/7dc7be48-8902-41f0-9b76-f09787ec9616">

**Comment by kiwisg124** [comment]
Looking at combinations of variables for my town .. when TechChangeStatus has **Eligible to Order** Alt Reason Code shows FTTP_SA . When a number of addresses become orderable the TechChangeStatus toggles from Build Finalised to Eligible to Order.

**Comment by kiwisg124** [comment]
For my street and town the following is evident.

Tech change status progress milestones roughly (same paths not shown for simplicity )

Not Planned -> In Design -> Committed -> Build Finalised -> Eligible to Order -> New Tech Connected -> Previous Tech Disconnected

During the progress from Committed to Build Finalised the targetEligibilityQuarter appears to be blanked out

Hopefully that does not indicate further delays.

**Comment by kiwisg124** [comment]
After Build Finalised and before Eligible to Order set could see the future orderable date here https://www.telstra.com.au/internet/nbn-fibre-upgrade

**Comment by LukePrior** [comment]
Hey,

Thanks for your work on investing this, unfortunately I have limited capability to work on non-vital issues for this project currently.

If you would like to open a PR with changes to the site and documentation that would be welcomed, otherwise I may revisit in future.
